products/monitoring/api.yaml

```diff
@@ -1623,6 +1623,16 @@ objects:
           name: content
           description: String or regex content to match (max 1024 bytes)
           required: true
+        - !ruby/object:Api::Type::Enum
+          name: matcher
+          description: The type of content matcher that will be applied to the server output,
+            compared to the content string when the check is run.
+          values:
+            - :CONTAINS_STRING
+            - :NOT_CONTAINS_STRING
+            - :MATCHES_REGEX
+            - :NON_MATCHES_REGEX
+          required: true
```

Comment:
Does this field need to be `required`? Adding a required field to a resource that already exists will break existing user configs, as nobody has this defined right now

Reply:
Got it. I will removed as its an optional field